lib/WP_Auth0_DBManager.php

```diff
@@ -92,56 +104,159 @@ public function install_db() {
 
 		if ( $this->current_db_version < 12 ) {
 
-      if ( strpos( $cdn_url, '10.' ) === false ) {
-        $options->set('use_lock_10', false);
-      } else {
-        $options->set('use_lock_10', true);
-      }
+			if ( strpos( $cdn_url, '10.' ) === false ) {
```

Comment:
I mentioned this but leaving it for other reviewers, in the future let's cover these changes in a separate internal PR, to keep the review process cleaner.

Reply:
@cocojoe - Understood. Can all formatting and minor stuff go together in a final cleanup PR or something? 

Most of the cases where there was formatting changes were part of a minor refactor. In this case, I moved all the vars we were using to the top of that method and reformatted a couple of the blocks.

Reply:
Yeah, it's not a priority. However, if you have some time and want to do a lint pass and also adopt some linting standards then it would be good to do. Although more likely this will be for the vNext when this can be standardised from the start.